rootfs-builder/suse/config.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This will also need an SPDX license header (unless you raise a PR to change https://github.com/kata-containers/tests/blob/master/.ci/static-checks.sh#L334 :)